pydata_google_auth/cache.py

```diff
@@ -123,6 +124,34 @@ def _save_user_account_credentials(credentials, credentials_path):
         logger.warning("Unable to save credentials.")
 
 
+def _load_service_account_credentials_from_file(credentials_path, **kwargs):
+    try:
+        with open(credentials_path) as credentials_file:
+            credentials_json = json.load(credentials_file)
+    except (IOError, ValueError) as exc:
+        logger.debug(
+            "Error loading credentials from {}: {}".format(credentials_path, str(exc))
+        )
+        return None
```

Comment:
Have you considered using [service_account.Credentials.from_service_account_file](https://googleapis.dev/python/google-auth/latest/reference/google.oauth2.service_account.html#google.oauth2.service_account.Credentials.from_service_account_file) instead of `from_service_account_info` so that you don't have to manually parse the JSON?

Reply:
This was because I went with a separate `_load_service_account_credentials_from_info` function that can process a dict instead of file, though that is also doable if you think that we should merge these functions

Reply:
Acknowledged. I think the `_info` function will actually be useful for a generic load any kind of credentials function (https://github.com/pydata/pydata-google-auth/issues/39#issuecomment-823302217) so I'm okay keeping this.